opencensus-shim/src/main/java/io/opentelemetry/api/trace/ProxyingSpan.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.trace;
```

Comment:
is this the right package for this?

Reply:
Good catch. No, this package is not correct as it is already used in the API itself.

Reply:
Should I move it over to the opencensusshim package or somewhere else?

Reply:
yes, please